dev-scripts/quick

```diff
@@ -43,6 +44,12 @@ BUILD_ARGS+=("--build-arg=CATTLE_FLEET_VERSION=${CATTLE_FLEET_VERSION}")
 BUILD_ARGS+=("--build-arg=CATTLE_HELM_VERSION=${CATTLE_HELM_VERSION}")
 BUILD_ARGS+=("--build-arg=RANCHER_TAG=${TAG}")
 BUILD_ARGS+=("--build-arg=RANCHER_REPO=${REPO}")
+if [ "$DEBUG" = true ]; then
+  # Remove -s (strip symbols)
+  BUILD_ARGS+=("--build-arg=LINKFLAGS=-extldflags -static")
+  # Disable compiler optimizations
+  BUILD_ARGS+=("--build-arg=EXTRAFLAGS=-gcflags=all=-N -l")
```

Comment:
Two suggestions:

1. Change the variable name to `GCFLAGS`. Right now, you can't add a non-gcflags flag to this because everything will be part of `-gcflags` (due to the quotes in the Dockerfile).
2. Do like we do with `-ldflags` for consistency, `-gcflags` is hardcoded in the command, then we can just add flags to it in `GCFLAGS`.

```suggestion
  BUILD_ARGS+=("--build-arg=GCFLAGS=all=-N -l")
```